reference-architecture/rhv-ansible/example/vars/ovirt-39-vars.yaml

```diff
@@ -70,15 +74,35 @@ node_vm:
 #    - sed -i 's@com.redhat.rhevm.vdsm@ovirt-guest-agent.0@' /etc/udev/rules.d/55-ovirt-guest-agent.rules
 #    - 'udevadm trigger --subsystem-match="virtio-ports"'
 
-cloud_init_script: |
+cloud_init_script_node: |
+  runcmd:
+    - mkdir -p '/var/lib/origin/openshift.local.volumes'
+    - /usr/sbin/mkfs.xfs -L localvol /dev/vdc
```

Comment:
@cwilkers :+1: 
1. I'm surprised there's no LVM layer on top - how can you later increase it if it's not large enough?
2. You can add '-K' to the mkfs.xfs command - there's no need to discard anything on the disk.